languages/python/sqlalchemy-oso/sqlalchemy_oso/partial.py

```diff
@@ -16,15 +16,9 @@
 EmitFunction = Callable[[Session, Any], Any]
 
 
-def partial_to_query(expression: Expression, session: Session, model) -> Query:
-    """Convert constraints in ``partial`` to a query over ``model``."""
-    query = session.query(model)
-
-    expr = translate_expr(expression, session, model)
-    if expr is not None:
-        return query.filter(expr)
-
-    return query
+def partial_to_filter(expression: Expression, session: Session, model):
```

Comment:
Is this function a useful abstraction?

Reply:
Yeah, we might rename & change the internals of this to match other frameworks.